### DISCUSSION TITLE
tj/ssh-xterminal

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#21] [Announcements]
# tj/ssh-xterminal

大家好，我是TJ君，一个专注推荐开源、免费、好用工具的程序员！ 今天给大家推荐一款好看又强大的SSH工具：Xterminal 。 下面给大家盘点下 Xterminal 中的重要特性，看看是否适合你的口味。

https://www.didispace.com/tj/ssh-xterminal.html

**Comment by ceozero** [comment]
好东西，有邀请码和优惠码没？准备充V。
